jplag/src/main/java/de/jplag/reporting/reportobject/ReportObjectFactory.java

```diff
@@ -64,36 +82,38 @@ private static OverviewReport generateOverviewReport(JPlagResult result) {
         overviewReport.setMetrics(getMetrics(result));
         overviewReport.setClusters(clusteringResultMapper.map(result));
 
-        return overviewReport;
+        fileWriter.saveAsJSON(overviewReport, path, "overview.json");
+
     }
 
-    /**
-     * Generates detailed ComparisonReport DTO for each comparison in a JPlagResult.
-     * @return A list with ComparisonReport DTOs.
-     */
-    private static List<ComparisonReport> generateComparisonReports(JPlagResult result) {
-        List<ComparisonReport> comparisons = new ArrayList<>();
-        getComparisons(result).forEach(comparison -> comparisons.add( //
-                new ComparisonReport(comparison.getFirstSubmission().getName(), //
-                        comparison.getSecondSubmission().getName(), //
-                        comparison.similarity(), //
-                        getFilesForSubmission(comparison.getFirstSubmission()), //
-                        getFilesForSubmission(comparison.getSecondSubmission()), //
-                        convertMatchesToReportMatches(result, comparison, comparison.getMatches()) //
-                )));
-        return comparisons;
+    private static void copySubmissionFilesToReport(String path, JPlagResult result) {
+        List<JPlagComparison> comparisons = result.getComparisons(result.getOptions().getMaximumNumberOfComparisons());
+        var submissions = getSubmissions(comparisons);
+        var submissionsPath = createDirectory(path, "Submissions");
```

Comment:
another magic string that could be put into a constant

Reply:
esp. the name should be lower case (at least from my point of view :) )